internal/dataplane/kongstate/util.go

```diff
@@ -211,14 +211,7 @@ func kongPluginFromK8SPlugin(
 	}, nil
 }
 
-var rawPatchPattern = `[{"op":"%s","path":"%s","value":%s}]`
-
-type JSONPatchOp string
-
-var (
-	JSONPatchOpAdd     JSONPatchOp = "add"
-	JSONPatchOpReplace JSONPatchOp = "replace"
-)
+var rawPatchPattern = `[{"op":"add","path":"%s","value":%s}]`
```

Comment:
this `json-patch` seems not be able to use `add` and empty path pointing to root to set entire object. Running `add` on an empty object with empty path will receive an empty object -- I think it should be an issue of the `json-patch` package. Now we could only use `replace` on path `""` to achieve fetching whole config from secret.